cli/tests/test_completion.rs

```diff
@@ -370,42 +549,40 @@ fn test_aliases_are_completed() {
         ])
         .success();
 
-    let mut test_env = test_env;
-    test_env.add_env_var("COMPLETE", "fish");
-    let test_env = test_env;
-    let work_dir = test_env.work_dir("repo");
-
-    let output = work_dir.run_jj(["--", "jj", "user-al"]);
-    insta::assert_snapshot!(output, @r"
-    user-alias
-    [EOF]
-    ");
+    let output = work_dir.complete_at(shell, 1, ["user-al"]);
+    match shell {
+        Shell::Bash | Shell::Zsh => {
+            insta::assert_snapshot!(output, @"user-alias[EOF]");
+        }
```

Comment:
either wrap in `insta::allow_duplicates! { .. }` or split match arms.

(you'll need to run tests without using `nextest` to detect this kind of problems.)